woocommerce/woocommerce-gutenberg-products-block
# Revert "Remove unused Add to Cart product element (#11948)"

This reverts PR https://github.com/woocommerce/woocommerce-blocks/pull/11948.

## What

Fixes woocommerce/woocommerce-blocks#12032.

## Why

For some reason, that PR was causing an incompatibility between the All Products block and the Mini-Cart block. I didn't investigate it further, but I propose to revert the PR and re-open https://github.com/woocommerce/woocommerce/issues/44503 so this doesn't block the 11.7.0 release.

## Testing Instructions

Note: the issue only happens when building with `npm run build`. It's not reproducible with `npm run start`.

1. Add the Mini-Cart block and the All Products block to the same post or page.
2. View the post in the frontend.
3. Open the Mini-Cart.
4. Verify there is no JS error and the Mini-Cart contents (in case you have added a product to your cart) can be seen.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
